deployment/docker-compose/template.core_backend.env

```diff
@@ -34,6 +34,11 @@ LITELLM_ENDPOINT="http://localhost:4000"
 #PGVECTOR_VECTOR_SIZE=1024
 
 #### Speech APIs ###############################################################
+# Set this variable to 'external' or 'custom' accordingly to toggle the /voice-search endpoint
+# By default it is not set so it defaults to None
+# TOGGLE_VOICE=external
```

Comment:
rename to `VOICE_SEARCH_MODE` for clarity? Toggle implies that it's on or off. Also I'd suggest changing the description to:

# This variable controls whether the voice search feature is inactive (commented out or set to None), active using external services ("external") or self-hosted APIs ("internal"). Default is disabled.